sctp/templates/_helpers.tpl

```diff
@@ -44,8 +44,33 @@ drax/technology: 5g
 drax/instanceId: "{{ tpl .Values.bootstrapId . }}"
 drax/redis-url: "{{ tpl .Values.redisConfig.hostname . }}"
 drax/redis-port: "{{ tpl .Values.redisConfig.port . }}"
+drax/helm-version: "{{ tpl .Values.global.helmVersion . }}"
+drax/5g-version: "{{ tpl .Values.global.tag . }}"
 {{- end }}
 
+{{/*
+Common labels
+*/}}
+{{- define "sctp.labels" -}}
```

Comment:
@ezeljkovic why do we need these new helper template? The other changes (amf-controller, cu-up) just have the two new labels added to the existing template? (In fact, they have also been added above here as well..)